addons/test_mail/tests/test_mail_followers_unsubscribe.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.test_mail.tests.test_message_post import TestMessagePostCommon
+from odoo.tests import tagged
+from odoo.tests import users
+
+
+@tagged('mail_followers', 'post_install', '-at_install')
+class UnfollowFromInboxTest(TestMessagePostCommon):
```

Comment:
Yes, you’re right, I don’t need the message because I can rely on message_post for creating it. I have removed the use of _notify_thread in favor of message_post that call it anyway. It simplifies the test, thanks.